dcpy/test/extract/resources/ingest_script.py

```diff
@@ -0,0 +1,7 @@
+from dcpy.extract import TMP_DIR
+
+
+def runner():
+    path = TMP_DIR / "t.txt"
```

Comment:
Should we use an actual temp dir for tests, distinct from what we'd actually use? (e.g. for local testing?)

Reply:
Addressed [here](https://github.com/NYCPlanning/data-engineering/pull/654/commits/989b49f1f8b43a31116b85bceda5e5457240545d)